src/Shared/CertificateGeneration/CertificateManager.cs

```diff
@@ -171,10 +196,23 @@ public X509Certificate2 CreateAspNetCoreHttpsDevelopmentCertificate(DateTimeOffs
                 pathLengthConstraint: 0,
                 critical: true);
 
+            byte[] bytePayload;
+
+            if (AspNetHttpsCertificateVersion != 0)
+            {
+                bytePayload = new byte[AspNetHttpsOidFriendlyNameLength + 1];
```

Comment:
Wasn't the idea to replace rather than append to the friendly name for versions > 0? Having the name and the version concatenated is weird, it would read as a control character if you decoded it as ascii.

Reply:
I mean we shouldn't be decoding it at all as ASCII anyways. It was only used for testing.

Reply:
Yeah, but a field still shouldn't contain two different types of data with different formats.